### PR TITLE
Proposed an editing workflow for #194

### PR DESCRIPTION
**Pull request by jaredmorgs**
I have tried this process and it works well, but the original instruction set left a few too many questions.
I've proposed a workflow that folks can use to continue using draw.io.

I have not published the file just in case changes are required.

### All Submissions:

* [x] Does your PR affect the documentation?
* [x] If yes, did you update the documentation or create an issue for updating it?

### Your first submission

* [ ] Have you added your name to the list of [contributors.adoc](https://github.com/docToolchain/docToolchain/blob/master/src/docs/manual/05_contributors.adoc)?

Not as yet. I will do that later on.

* [ ] Did you build the docs and copy them to the `/docs` folder?
* [ ] If not, did you create an issue for doing so?

I have not published the file just in case changes are required.

I have added a note into #194 to re-build the docs once the change is improved.

### CODE REVIEW

File: src/docs/manual/03_task_exportDrawIo.adoc [hunk 1]
```diff
@@ -1,15 +1,45 @@
...
+. In the Image modal, make sure that "Include a copy of my diagram" is selected.
+. Click Export to save the PNG file with the pattern `[file].dio.png`.
++
+NOTE: Specifying `.dio` (short for "**d**raw**io**") in the name will help you identify PNG files containing embedded XML diagram source.
++
+. Commit the exported PNG file to source control.
+
+You now have a diagram that can be managed in source control and edited using draw.io. 
+
+.Edit a diagram in Draw.io Offline editor mode
```
Comment (rdmueller) [inline src/docs/manual/03_task_exportDrawIo.adoc]: Do we need this section? I think it should be replaced with a hint to the Desktop version: https://about.draw.io/integrations/#integrations_offline

With this version, you can edit your diagrams offline and don't have to take care of duplicate files.

**Comment by rdmueller** [comment]
Thanx! This is great! Better documentation is very valuable.

Two remarks: 
- I initially came up with the extension `.dio` for "drawio". As I do understand you used `.dia` for "diagram". I hope it is ok that I switched to `dio` and I added an explanation
- see my comment for the Desktop version. I think it solves the problems of the offline mode.

**Comment by jaredmorgs** [comment]
Great pick up on the dio vs dia, and my apologies for confusing those two.
It's a really good pattern to use to identify these files.

Good tip about the desktop editor as well for draw.io. I'm going to give it
a try in the workflow where I work and see how it performs.

Did you want me to adjust the PR with the new instructions about the
desktop editor unless it's already been merged?

File: src/docs/manual/03_task_exportDrawIo.adoc [hunk 2]
```diff
@@ -1,6 +1,6 @@
 :filename: manual/03_task_exportDrawIo.adoc
 ifndef::imagesdir[:imagesdir: ../images]
-:experimental:
```
Comment (jaredmorgs) [inline src/docs/manual/03_task_exportDrawIo.adoc]: :experimental: gives you access to the Menu and keyboard macros which I've used on line 18. Without that macro, the keyboard macro won't render correctly.

File: src/docs/manual/03_task_exportDrawIo.adoc (hunk 1 above)
Comment (jaredmorgs) [reply src/docs/manual/03_task_exportDrawIo.adoc]: This is a much better idea. I just installed the Windows desktop version and its heaps better than the offline web editor. I was just going with what I thought was suggested in the original text.

File: src/docs/manual/03_task_exportDrawIo.adoc (hunk 2 above)
Comment (rdmueller) [reply src/docs/manual/03_task_exportDrawIo.adoc]: ah. ok. didn't see that. let's put it back in...

Comment (rdmueller) [reply src/docs/manual/03_task_exportDrawIo.adoc]: ah - I've seen you've already put it back in. perfect

**Comment by rdmueller** [comment]
> Did you want me to adjust the PR with the new instructions about the
> desktop editor unless it's already been merged?

I think best would be if you first check out the Desktop version of draw.io. And then let's see if we still need the section about offline mode (IMHO, the Desktop version is better to use).

**Comment by jaredmorgs** [comment]
Yep already done it and I agree the Desktop version is superior. I adjusted
the procedure accordingly.

**Comment by rdmueller** [comment]
perfect! Welcome to the ever growing lsit of contributors!
Feel free to add yourself to the list of contributors within the docs through another PR.

**Pull request merged**